src/hotspot/share/adlc/output_c.cpp

```diff
@@ -3249,15 +3212,12 @@ void ArchDesc::defineClasses(FILE *fp) {
       definePeephole(_CPP_PEEPHOLE_file._fp, instr);
 
     // Output code to convert to the cisc version, if applicable
-    used |= instr->define_cisc_version(*this, fp);
+    instr->define_cisc_version(*this, fp);
 
     // Output code to convert to the short branch version, if applicable
-    used |= instr->define_short_branch_methods(*this, fp);
+    instr->define_short_branch_methods(*this, fp);
```

Comment:
Should you update these 2 methods since their return value is not used anymore anywhere?

Reply:
Good point, done, thanks!